### DISCUSSION TITLE
Update Git Project Manager Details on VSCode to include better path examples

### DISCUSSION THREAD

**Discussion by Amnesiac9** [alefragnani/vscode-project-manager#686] [Q&A] [answered]
Hi, I'm pretty new to coding, just started learning this year. I had some trouble with configuration at first, but I finally figured it out. I thought it might be a good idea to update the "details" with this small change to avoid confusion for new programmers like me.

 I found this part confusing, because I don't have much experience with lunix (I'm assuming that /home/ is a lunix thing): 

```
Before start using GPM you need to configure the base folders that the extension will search for git repositories. Edit settings.json from the File -> Preferences -> Settings and add the following config


{
    "gitProjectManager.baseProjectsFolders": [
        "/home/user/nodeProjects",
        "/home/user/personal/pocs"
    ]
}
```

This example in the documentation was much better for figuring out how to format my file paths, and the ignore paths. I thought it would be a good idea to include it in the details on VSCode. 

```
    "projectManager.git.baseFolders": [
        "c:\\Projects\\code",
        "d:\\MoreProjects\\code-testing",
        "$home\\personal-coding"
    ]

    Define the folders which contains the projects

    "projectManager.git.ignoredFolders": [
        "node_modules", 
        "out", 
        "typings", 
        "test"
        "fork*"
    ],

    Define which folders should be ignored (inside the BaseFolders).
```

**Comment by alefragnani** [comment, marked as answer]
Hi @Amnesiac9 ,

I guess you reportedthe issue in the wrong extension 😬 . This one is **Project Manager**, not **Git Project Manager**, which the repo is https://github.com/felipecaputo/git-project-manager.

Hope this helps

**Reply by Amnesiac9** [reply]
Oof, sorry about that!

**Reply by alefragnani** [reply]
That's ok. No need to be sorry 👍
